sdk/core/abort-controller/package.json

```diff
@@ -58,16 +58,16 @@
     "build:samples": "echo Obsolete",
     "build:test": "npm run clean && tshy && dev-tool run build-test",
     "build": "npm run clean && tshy && api-extractor run --local",
-    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.{ts,cts,mts}\" \"test/**/*.{ts,cts,mts}\" \"*.{js,mjs,cjs,json}\"",
```

Comment:
can you file a core backlog item to consider moving this into a dev-tool command along with format/lint/lint:fix? Feels like it's going to be a pain to keep having to update so many options inline like this

Reply:
Opened https://github.com/Azure/azure-sdk-for-js/issues/28660